tests/src/test/java/alluxio/stress/StressMasterBenchIntegrationTest.java

```diff
@@ -204,6 +209,76 @@ private void validateTheResultWithWriteType(String writeType) throws Exception {
     assertTrue(summary2.collectErrorsFromAllNodes().isEmpty());
   }
 
+  @Test
+  public void writeTypeALLTaskTest() throws Exception {
+    // redirect the output stream
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    PrintStream originalOut = System.out;
+    System.setOut(new PrintStream(out));
+
+    new StressMasterBench().run(new String[] {
+        "--in-process",
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/",
+        "--operation", "CreateFile",
+        "--fixed-count", "20",
+        "--target-throughput", "300",
+        "--threads", "5",
+        "--warmup", "0s", "--duration", "3s",
+        "--write-type", "ALL",
+    });
+
+    String printOutResult = out.toString();
+    List<MasterBenchSummary> resultList = getJsonResult(printOutResult);
+
+    assertEquals(resultList.size(), 4);
+
+    // the possible write types
+    List<String> writeTypes = ImmutableList.of("MUST_CACHE", "CACHE_THROUGH",
+        "ASYNC_THROUGH", "THROUGH");
+    for (int i = 0; i < resultList.size(); i++) {
+      MasterBenchSummary summary = resultList.get(i);
+      // confirm that the task was executed with certain write type and output no errors
+      assertEquals(summary.getParameters().mWriteType, writeTypes.get(i));
+      assertFalse(summary.getNodeResults().isEmpty());
+      assertTrue(summary.collectErrorsFromAllNodes().isEmpty());
+    }
+
+    // reset the output to the console
+    System.setOut(originalOut);
+  }
+
+  private List<MasterBenchSummary> getJsonResult(String output) throws Exception {
```

Comment:
remove duplicate code. Move this method into `StressBenchIntegrationTestUtils.java`